bin/audit-svg-colors.js

```diff
@@ -16,36 +16,69 @@ const { readFileSync } = require( 'fs' );
 const path = require( 'path' );
 const PALETTE = require( '@automattic/color-studio' );
 const chroma = require( 'chroma-js' );
-const _ = require( 'lodash' );
+
+/**
+ * Native Sort - replacement of ._sortBy Lodash function
+ *
+ * @returns Sorted array.
+ */
+const sortBy = () => {
+	return ( objA, objB ) => {
+		if ( objA.to.name > objB.to.name ) {
+			return 1;
+		} else if ( objB.to.name > objA.to.name ) {
+			return -1;
+		}
+		return 0;
+	};
+};
```

Comment:
I think we can unwrap this function since we'll always need to invoke it to get the actual sorting function as a result:

```suggestion
const compareByName = ( objA, objB ) => {
	if ( objA.to.name > objB.to.name ) {
		return 1;
	} else if ( objB.to.name > objA.to.name ) {
		return -1;
	}
	return 0;
};
```

As you can see, we also renamed the function, since it's now a comparator function.

Alternatively, we could use [`String.prototype.localeCompare()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) since it is supported in Node.js.